{{cookiecutter.repo_name}}/.gitignore

```diff
@@ -92,3 +92,9 @@ docs/_build/
 
 # PyBuilder
 target/
+Dockerfile
```

Comment:
Looks like we want these as well as:
```
docker-base/*
!docker-base/build.sh
!docker-base/Dockerfile
```

Also - whitespace after `.project_name`.